.github/workflows/ci-release.yml

```diff
@@ -34,7 +38,31 @@ jobs:
           SONATYPE_USER: ${{ secrets.SONATYPE_USER }}
           SONATYPE_TOKEN: ${{ secrets.SONATYPE_TOKEN }}
           CI: true
-        run: ./gradlew --parallel --max-workers=4 -PreleaseBuild=true clean check && ./gradlew -PreleaseBuild=true publish
+        run: |
+          # Build arguments to feed to the single gradlew publish command
+          if [ "${{ matrix.java }}" = "8" ]; then
```

Comment:
I was able to test the `"${{ matrix.java }}"` part in my local fork: https://github.com/Scottmitch/servicetalk-1/runs/2028750913

Reply:
In the output of your build I see:
```
FIRST_GRADLE_TARGETS: FIRST_GRADLE_TARGETS
SECOND_GRADLE_TARGETS:  :servicetalk-concurrent-jdkflow:publish
```
Was the `FIRST_GRADLE_TARGETS` not set correctly?

Reply:
`echo "FIRST_GRADLE_TARGETS: FIRST_GRADLE_TARGETS"` -> I had a typo where I forgot the `$` on the variable, but the second line shows the state is being set.